src/translations/translations.fr.js

```diff
@@ -62,7 +62,9 @@ export const translations = {
   'cc-addon-admin.admin': `Administration de l'add-on`,
   'cc-addon-admin.danger-zone': `Zone de danger`,
   'cc-addon-admin.delete': `Supprimer l'add-on`,
-  'cc-addon-admin.delete-description': () => sanitize`La machine virtuelle sera arrêtée dans 24 heures.<br>Les backups seront gardés suivant la politique de rétention.<br>Supprimer cet add-on le rendra directement indisponible.`,
+  'cc-addon-admin.delete-backups': `Les sauvegardes seront gardées suivant la politique de rétention.`,
+  'cc-addon-admin.delete-disclaimer': `Supprimer cet add-on le rend immédiatement indisponible.`,
```

Comment:
question: for both english and french we use the future for the two other sentences options but not here, is there a reason why? 
I was asking myself this question as it is a consequence of clicking on the button. And that action will be in the future if you're reading the text. :thinking:

Reply:
I'm fine with the current form it was more of a thought than a question :wink: :hugs: